src/js/target.js

```diff
@@ -1,28 +1,22 @@
 class Target {
 	constructor(targetEl) {
 		this.targetEl = targetEl;
-		this.rectObject = targetEl.getBoundingClientRect();
-	}
-
-	getEdge(edge){
```

Comment:
This shouldn't really be removed without a major bump, I know it's not used in the codebase but we can't guarantee that a downstream user isn't relying on it unfortunately 😔

I suggest re-adding it but changing to be deprecated (according to our [deprecation guidelines](http://origami.ft.com/docs/component-spec/modules/#deprecation-of-a-module-feature)):

```js
// @deprecated This is not used anywhere in the codebase, seems like we don't need it
getEdge(edge) {
	console.warn('The `getEdge` method is deprecated and will be removed in the next major version of o-tooltip');
	return this.targetEl.getBoundingClientRect()[edge];
}
// @deprecated ^^^
```

Reply:
@rowanmanning Ok yeah my bad about removing that code, went a bit rogue, thought the target was only used internally by the tooltip class, didn't see you were exposing it to consumers. Cleared these up now.